core/src/raw/http_util/multipart.rs

```diff
@@ -281,9 +404,12 @@ impl MixedPart {
 
         let (parts, body) = req.into_parts();
 
-        let content = match body {
-            AsyncBody::Empty => Bytes::new(),
-            AsyncBody::Bytes(bs) => bs,
+        let (content_length, content) = match body {
+            AsyncBody::Empty => (0, None),
+            AsyncBody::Bytes(bs) => (
+                bs.len() as u64,
+                Some(Box::new(oio::Cursor::from(bs)) as Streamer),
+            ),
             AsyncBody::Stream(_) => panic!("multipart request can't contain stream body"),
```

Comment:
Maybe we can make multipart request contain stream body now.

Reply:
Not part of this PR.